src/main/java/org/opensearch/timeseries/util/TimeSeriesSafeSecurityInjector.java

```diff
@@ -48,21 +57,21 @@ public void injectUserRolesFromDetector(ActionListener<Void> injectListener) {
             return;
         }
 
-        ActionListener<Optional<AnomalyDetector>> getDetectorListener = ActionListener.wrap(detectorOp -> {
+        ActionListener<Optional<? extends Config>> getDetectorListener = ActionListener.wrap(detectorOp -> {
```

Comment:
nit: detectorListener can be changed to config or timeseries listener?

Reply:
changed